Extensions/TopDownMovementBehavior/topdownmovementruntimebehavior.ts

```diff
@@ -343,7 +343,7 @@ namespace gdjs {
         direction = topDownMovementHook.overrideDirection(direction);
       }
       for (const topDownMovementHook of this._topDownMovementHooks) {
-        topDownMovementHook.beforeSpeedUpdate();
+        topDownMovementHook.beforeSpeedUpdate(direction);
```

Comment:
Is there a risk another parameter would be passed in the future? If yes should you pass an object?

Reply:
I don't know, I guess we could pass a context to be safe.